x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/indices.freeze/10_basic.yml

```diff
@@ -1,5 +1,8 @@
 ---
 "Basic":
+- skip:
+    features: [ "allowed_warnings" ]
```

Comment:
Should this be "warnings" instead of "allowed_warnings" ? The difference is that if you only allow, then the warning can be missing and still pass the test. "warnings" is more like "require_warnings"